gym/utils/env_checker.py

```diff
@@ -168,16 +198,18 @@ def _check_normalized_action(action_space: spaces.Box):
         )
 
 
-def _check_returned_values(
-    env: gym.Env, observation_space: spaces.Space, action_space: spaces.Space
-) -> None:
-    """
-    Check the returned values by the env when calling `.reset()` or `.step()` methods.
+def _check_returned_values(env: gym.Env, observation_space: Space, action_space: Space):
+    """Check the returned values by the env when calling `.reset()` or `.step()` methods.
```

Comment:
```suggestion
    """Check the returned values by the env when calling :meth:`env.reset()` or :meth:`env.step()` methods.
```

Reply:
Added but without the extra brackets, I think :meth: does that itself